### DISCUSSION TITLE
Bot error

### DISCUSSION THREAD

**Discussion by aternos24h** [sleeyax/aternos-discord-bot#43] [General]
Hi, I created a bot on your replit template.
Everything works fine, but when I want to turn on the server, it pops up: Access forbidden. Please try again later
![Zrzut ekranu 2022-05-25 111205](https://user-images.githubusercontent.com/87474945/170226927-71b886e5-9608-41c5-81cb-c3530e661316.png)
.

**Comment by sleeyax** [comment]
See https://github.com/sleeyax/aternos-discord-bot/issues/42#issuecomment-1130240988

**Reply by aternos24h** [reply]
is it payable?

**Reply by sleeyax** [reply]
Wym Payable? Just get a cheap vps from wishosting, servercheap, contabo etc. countless of options just put in a little effort to google it.

**Reply by aternos24h** [reply]
Thank you for your hel
![Zrzut ekranu 2022-05-25 133157](https://user-images.githubusercontent.com/87474945/170252633-2f3a9d79-ee72-4728-98ef-2459de02682e.png)
p
